.github/actions/constellation_measure/action.yml

```diff
@@ -51,13 +51,33 @@ runs:
       run: |
         KUBECONFIG="$PWD/constellation-admin.conf" kubectl rollout status ds/verification-service -n kube-system --timeout=3m
         CONSTELL_IP=$(jq -r ".ip" constellation-id.json)
-        pcr-reader --constell-ip ${CONSTELL_IP} -format yaml > measurements.yaml
+        # pcr-reader --constell-ip ${CONSTELL_IP} -format yaml > measurements.yaml
+        pcr-reader --constell-ip ${CONSTELL_IP} -format json > measurements.json
         case $CSP in
           azure)
-            yq e 'del(.[0,6,10,11,12,13,14,15,16,17,18,19,20,21,22,23])' -i measurements.yaml
+            yq e 'del(.[0,6,10,16,17,18,19,20,21,22,23])' -I 0 -o json -i measurements.json
+            yq '.4.warnOnly = false |
+              .8.warnOnly = false |
+              .9.warnOnly = false |
+              .11.warnOnly = false |
+              .12.warnOnly = false |
+              .13.warnOnly = false |
+              .15.warnOnly = false |
+              .15.expected = "0000000000000000000000000000000000000000000000000000000000000000"' \
+              -I 0 -o json -i measurements.json
             ;;
           gcp)
-            yq e 'del(.[11,12,13,14,15,16,17,18,19,20,21,22,23])' -i measurements.yaml
+            yq e 'del(.[16,17,18,19,20,21,22,23])' -I 0 -o json -i measurements.json
+            yq '.0.warnOnly = false |
+              .4.warnOnly = false |
+              .8.warnOnly = false |
+              .9.warnOnly = false |
+              .11.warnOnly = false |
+              .12.warnOnly = false |
+              .13.warnOnly = false |
+              .15.warnOnly = false |
+              .15.expected = "0000000000000000000000000000000000000000000000000000000000000000"' \
+              -I 0 -o json -i measurements.json
```

Comment:
I think if we already merge the CI change to JSON as part of this PR, we would also need to adjust fetch-measurements (which currently expects .yaml URLs).

Reply:
I would opt to not update the `fetch-measurements` command default URLs until the new measurement pipeline is implemented.
`fetch-measurements` only fetches the signed measuremnts for release images, of which we have non in json format until the next release

Reply:
I was also thinking about this, but then:

1. Having JSON and the new structure in there for parsing in general would already be part of the new measurement pipeline and help development; it wouldn't be wasted work (unless I am missing something here).
2. The new measurement pipeline would only add another JSON object holding metadata and change the URLs depending on the image UID, so we likely would mainly build stuff on top of what is currently there (and tweak the download URLs).
3. In a strict sense, this PR would break the concept of having a "releasable main" since you adjust the CI (which will likely look completely different with the new pipeline) but don't make adjustments to the downloading code.

Reply:
I think there was some miscommunication here.
When I said I would like to not make changes to `fetch-measurements`, I was explicitly talking about the code of that command, more specifically the URL parsing, since that still uses the old public bucket.

I have since adapted the code to be agnostic to yaml/json formating when parsing measurements during `*M.FetchAndVerify()`